pkg/plugins/gateway/algorithms/vtc/token_estimator.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2024 The Aibrix Team.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vtc
+
+import (
+	"math"
+	"strings"
+)
+
+type SimpleTokenEstimator struct {
+	CharactersPerToken float64
+	OutputRatio        float64
+}
+
+func NewSimpleTokenEstimator() *SimpleTokenEstimator {
+	return &SimpleTokenEstimator{
+		CharactersPerToken: 4.0, // Rough approximation: 4 chars per token
```

Comment:
We see many cases have tokenizer dependency to give much more accurate results. We may change this part later once we decide to use a fast tokenizer or offload to tokenizer service

Reply:
yes, this is a simple estimator to get started with fariness based routing. not intended for long-term use.